psi4/src/psi4/libscf_solver/hf.cc

```diff
@@ -771,210 +501,189 @@ void HF::find_occupation()
     frac();
 }
 
-void HF::print_header()
-{
+void HF::print_header() {
     int nthread = 1;
-    #ifdef _OPENMP
-        nthread = Process::environment.get_n_threads();
-    #endif
-
-
-    outfile->Printf( "\n");
-    outfile->Printf( "         ---------------------------------------------------------\n");
-    outfile->Printf( "                                   SCF\n");
-    outfile->Printf( "            by Justin Turney, Rob Parrish, Andy Simmonett\n");
-    outfile->Printf( "                             and Daniel Smith\n");
-    outfile->Printf( "                             %4s Reference\n", options_.get_str("REFERENCE").c_str());
-    outfile->Printf( "                      %3d Threads, %6ld MiB Core\n", nthread, memory_ / 1048576L);
-    outfile->Printf( "         ---------------------------------------------------------\n");
-    outfile->Printf( "\n");
-    outfile->Printf( "  ==> Geometry <==\n\n");
+#ifdef _OPENMP
+    nthread = Process::environment.get_n_threads();
+#endif
 
+    outfile->Printf("\n");
+    outfile->Printf("         ---------------------------------------------------------\n");
+    outfile->Printf("                                   SCF\n");
+    outfile->Printf("            by Justin Turney, Rob Parrish, Andy Simmonett\n");
```

Comment:
This overhaul is more than enough justification to add your name to the list of authors of this chunk of code, Lori.  Especially with the number of times you've had to rebase this branch!  If we're going to continue the Psi3 convention of tagging the code segments, it may as well be accurate

Reply:
+1 here

Reply:
:+1: